packages/paste-libraries/clipboard-copy/stories/index.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import type {Story, Meta} from '@storybook/react';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Input} from '@twilio-paste/input';
+import {Label} from '@twilio-paste/label';
+import {Stack} from '@twilio-paste/stack';
+import {useUID} from '@twilio-paste/uid-library';
+
+import {useClipboard} from '../src';
+import type {UseClipboardProps} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Libraries/clipboard-copy',
+} as Meta;
+
+const Template: Story<{clipboardProps: UseClipboardProps}> = ({clipboardProps}) => {
+  const clipboard = useClipboard(clipboardProps);
+  const uid = useUID();
+
+  return (
+    <Stack orientation="vertical" spacing="space50">
+      <Box>
+        <Label htmlFor={uid}>SID</Label>
+        <Input ref={clipboard.target} id={uid} type="text" readOnly value="AC00000000000000000000000000" />
+      </Box>
+      <Button variant="primary" onClick={clipboard.copy}>
+        Copy
+      </Button>
+    </Stack>
+  );
+};
+
+export const Default = Template.bind({});
+
+Default.args = {};
+
+export const Timeout = Template.bind({});
```

Comment:
The only thing I might say about this one, is that from a visual and interactive perspective, it doesn't really do anything differently from the default story, from the users perspective. It kind of looks just the same.

Not sure if you can add something to it that illustrates the timeOut? If not this might be just fine to omit and we add a more detailed example when we get to the docs.

Reply:
Adding conditional copy state that times out:

![image](https://user-images.githubusercontent.com/6936736/130528283-b4809e74-ba43-4827-b0c9-57e89a308b9b.png)

![image](https://user-images.githubusercontent.com/6936736/130528272-8cca0899-8e54-4055-88f2-d33d22d4b1e1.png)

Reply:
Perfect